### PR TITLE
Profile Employee

### PR DESCRIPTION
**Pull request by ChinIndival**

### CODE REVIEW

**Comment by ChinIndival** [comment]
ready

**Comment by ChinIndival** [comment]
ready

File: resources/views/employees/profile/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+                        <div class="card-body">
+                            <div class="panel-body">
+                                <div class="row">
+                                    <div class="col-lg-6">
+                                        <div class="avata-name">
+                                        <legend class="avata text-center">{{ __('avatar') }}</legend>
+                                        <hr>
+                                        </div>
+                                        <div class="avata text-center">
+                                        <img src="{!! asset(config('app.link_avatar').$employee->avatar) !!}" class="avata-img img-circle" alt="">
```
Comment (BunnyPi04) [inline resources/views/employees/profile/index.blade.php]: convention

**Comment by ChinIndival** [comment]
ready

**Pull request merged**